eng/common/tools.ps1

```diff
@@ -93,10 +93,6 @@ function Exec-Process([string]$command, [string]$commandArgs) {
 }
 
 function InitializeDotNetCli([bool]$install) {
-  if (Test-Path variable:global:_DotNetInstallDir) {
```

Comment:
Won't this be overwritten?

Reply:
@marek-safar I don't know why but with this line if I change version in `global.json` - it simply returns old version and does nothing. And don't worry without this like it won't download SDK every hit.

Reply:
cc @ViktorHofer

Reply:
the .sh version works without changes as expected.

Reply:
I guess you could upstream this change

Reply:
But not sure how, afair the whole eng/** folder is autogenerated (by darc?)

Reply:
I think it lives in the normal repo at https://github.com/dotnet/arcade